cmd/validator/flags/flags.go

```diff
@@ -171,121 +172,121 @@ var (
-			"a voluntary exit",
+		Usage: "Comma separated list of public key hex strings to specify on which validator accounts to perform " +
+			"a voluntary exit.",
 		Value: "",
 	}
 	// ExitAllFlag allows stakers to select all validating keys for exit. This will still require the staker
 	// to confirm a userprompt for this action given it is a dangerous one.
 	ExitAllFlag = &cli.BoolFlag{
 		Name:  "exit-all",
-		Usage: "Exit all validators. This will still require the staker to confirm a userprompt for the action",
+		Usage: "Exits all validators. This will still require the staker to confirm a userprompt for the action.",
 	}
 	// ForceExitFlag to exit without displaying the confirmation prompt.
 	ForceExitFlag = &cli.BoolFlag{
 		Name:  "force-exit",
-		Usage: "Exit without displaying the confirmation prompt",
+		Usage: "Exits without displaying the confirmation prompt.",
 	}
 	VoluntaryExitJSONOutputPath = &cli.StringFlag{
 		Name: "exit-json-output-dir",
-		Usage: "The output directory to write voluntary exits as individual unencrypted JSON " +
+		Usage: "Output directory to write voluntary exits as individual unencrypted JSON " +
 			"files. If this flag is provided, voluntary exits will be written to the provided " +
 			"directory and will not be broadcasted.",
 	}
 	// BackupPasswordFile for encrypting accounts a user wishes to back up.
 	BackupPasswordFile = &cli.StringFlag{
 		Name:  "backup-password-file",
-		Usage: "Path to a plain-text, .txt file containing the desired password for your backed up accounts",
+		Usage: "Path to a plain-text, .txt file containing the desired password for your backed up accounts.",
 		Value: "",
 	}
 	// BackupDirFlag defines the path for the zip backup of the wallet will be created.
 	BackupDirFlag = &cli.StringFlag{
 		Name:  "backup-dir",
-		Usage: "Path to a directory where accounts will be backed up into a zip file",
+		Usage: "Path to a directory where accounts will be backed up into a zip file.",
 		Value: DefaultValidatorDir(),
 	}
 	// SlashingProtectionJSONFileFlag is used to enter the file path of the slashing protection JSON.
 	SlashingProtectionJSONFileFlag = &cli.StringFlag{
 		Name:  "slashing-protection-json-file",
-		Usage: "Path to an EIP-3076 compliant JSON file containing a user's slashing protection history",
+		Usage: "Path to an EIP-3076 compliant JSON file containing a user's slashing protection history.",
 	}
 	// KeysDirFlag defines the path for a directory where keystores to be imported at stored.
 	KeysDirFlag = &cli.StringFlag{
 		Name:  "keys-dir",
-		Usage: "Path to a directory where keystores to be imported are stored",
+		Usage: "Path to a directory where keystores to be imported are stored.",
 	}
 
 	// RemoteSignerCertPathFlag defines the path to a client.crt file for a wallet to connect to
 	// a secure signer via TLS and gRPC.
 	RemoteSignerCertPathFlag = &cli.StringFlag{
```

Comment:
we should review if we even need these anymore, we removed the Remote signer but since some of our internal APIs (gRPC) code might be still using it we should double-check if the flags are still needed.

Reply:
Yes I agree.
(This PR really focuses on "cosmetic" changes, without trying to understand/fix the underlying features under the flags.)